docs/application/native/guides/app-management/app-controls.md

```diff
@@ -664,6 +668,120 @@ create_base_gui(appdata_s *ad)
    }
    ```
 
+<a name="action"></a>
+## Handle Application Control Actions
+
+You can define and handle each application control action.
```

Comment:
@hjhun Please put a colon at the of the sentence as it following by bullet points. Also it will give consistency to the page. So kindly make the change as "... each application control action:"